web/src/components/FilingsCalendar.tsx

```diff
@@ -96,6 +107,20 @@ export const FilingsCalendar = (props: Props): ReactElement => {
             </div>
           );
         })}
+        {sortedFilteredOperatingFilings.length > listViewMoreIncrement && (
+          <div
+            className={"view-more-button"}
+            onClick={() => setNumberOfVisibleCalendarEntries((previous) => previous + listViewMoreIncrement)}
+            onKeyDown={() =>
+              setNumberOfVisibleCalendarEntries((previous) => previous + listViewMoreIncrement)
+            }
+            role="button"
+            tabIndex={0}
+            aria-label={`View ${listViewMoreIncrement} More Calendar Entries`}
+          >
+            View More
```

Comment:
All plaintext in the application is configured so that it can be externally edited in the CMS. We can pair and I can show you how to add this to the config for CMS. but as a rule of thumb, there should never be plaintext words in the JSX